src/sfctl/custom_cluster_upgrade.py

```diff
@@ -142,7 +144,8 @@ def sa_configuration_upgrade( #pylint: disable=missing-docstring,invalid-name,to
         max_percent_unhealthy_applications=unhealthy_applications,
         max_percent_unhealthy_nodes=unhealthy_nodes,
         max_percent_delta_unhealthy_nodes=delta_unhealthy_nodes,
-        max_percent_upgrade_domain_delta_unhealthy_nodes=upgrade_domain_delta_unhealthy_nodes) #pylint: disable=line-too-long
+        max_percent_upgrade_domain_delta_unhealthy_nodes=upgrade_domain_delta_unhealthy_nodes, #pylint: disable=line-too-long
```

Comment:
Don't disable line too long, use different variable names.

Reply:
I didn't.. this is preexisting

Reply:
Oh gotchya, okay